Mathlib/Init/Data/List/Basic.lean

```diff
@@ -26,12 +26,12 @@ open Option Nat
 #align list.nth List.get?
 
 /-- nth element of a list `l` given `n < l.length`. -/
-@[deprecated get]
+@[deprecated get] -- 2023-01-05
 def nthLe (l : List α) (n) (h : n < l.length) : α := get l ⟨n, h⟩
 #align list.nth_le List.nthLe
 
 set_option linter.deprecated false in
-@[deprecated]
+@[deprecated] -- 2023-01-05
```

Comment:
I think anything that's been deprecated for > 6 months is fair game to remove

Reply:
I wasn't sure if this one was still used. (There are *many* uses of deprecated lists stuff still.) Let me check...

Reply:
(That PR is now ready for review :-))